src/Components/Components/src/ComponentBase.cs

```diff
@@ -25,11 +25,6 @@ namespace Microsoft.AspNetCore.Components
     /// </summary>
     public abstract class ComponentBase : IComponent, IHandleEvent, IHandleAfterRender
     {
-        /// <summary>
-        /// Specifies the name of the <see cref="RenderTree"/>-building method.
-        /// </summary>
-        public const string BuildRenderTreeMethodName = nameof(BuildRenderTree);
```

Comment:
This was unused, as far as I can tell.

Reply:
yep. this was here is a very early iteration to share with the compiler